Assets/Scripts/Trigger/buttonTrigger.cs

```diff
@@ -42,5 +42,20 @@ public class buttonTrigger : Trigger {
 		}
 	}
 
+	public override void interact()
+	{
+		UITrigger secretItem = GetComponent<UITrigger> ();
+		if (secretItem != null && !secretItem.found) {
+			secretItem.interact ();
+			buttonPressed = secretItem.replacementSprites [0];
```

Comment:
I don't think it make sense to make a whole separate child class for this, and it doesn't affect the behaviour of button trigger in any way as ur not supposed to be able to "interact" with it at the first place